pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -92,6 +92,9 @@ type PrometheusSpec struct {
 
 	// +optional
 	Config *PrometheusConfiguration `json:"config,omitempty"`
+
+	// +optional
+	AdditionalContainers []corev1.Container `json:"additionalContainers,omitempty"`
```

Comment:
I think the `AdditionalContainers` should be placed under `TidbMonitorSpec` instead of `PrometheusSpec`